### PR TITLE
Change migration messages to be better for the current migration, PLA…

### PR DESCRIPTION
**Pull request by abador**
…TFORM-4807

### CODE REVIEW

File: extensions/wikia/FandomComMigration/FandomComMigration.i18n.php [hunk 1]
```diff
@@ -2,73 +2,6 @@
...
-
-$messages['zh-hans'] = array(
-	'fandom-com-migration-after' => '请注意：我们已经将您的社区域名迁移到了fandom.com。您可以在[http://zh.community.wikia.com/wiki/Help:Fandom%E5%9F%9F%E5%90%8D%E8%BF%81%E7%A7%BB Fandom帮助页]查找更多信息。',
-	'fandom-com-migration-before' => '请注意：在接下来的几个星期内，我们将会把您的社区域名迁移到fandom.com。您可以在[http://zh.community.wikia.com/wiki/Help:Fandom%E5%9F%9F%E5%90%8D%E8%BF%81%E7%A7%BB Fandom帮助页]查找更多信息。',
-);
-
-$messages['zh-hant'] = array(
-	'fandom-com-migration-after' => '请注意：我们已经将您的社区域名迁移到了fandom.com。您可以在[http://zh.community.wikia.com/wiki/Help:Fandom%E5%9F%9F%E5%90%8D%E8%BF%81%E7%A7%BB Fandom帮助页]查找更多信息。',
-	'fandom-com-migration-before' => '请注意：在接下来的几个星期内，我们将会把您的社区域名迁移到fandom.com。您可以在[http://zh.community.wikia.com/wiki/Help:Fandom%E5%9F%9F%E5%90%8D%E8%BF%81%E7%A7%BB Fandom帮助页]查找更多信息。',
+	'fandom-com-migration-before' => "Attention: Over the next few weeks, we are migrating your community to the Unified Community Platform. Find out more on [https://community.fandom.com/wiki/Help:Unified_Community_Platform] this Fandom help page.",
```
Comment (themech) [inline extensions/wikia/FandomComMigration/FandomComMigration.i18n.php]: Should we reuse the same message? something ucp related would be more appropriate. 
I'm not an expert in i18, but when reusing the message don't we risk downloading translations for the previous string?

File: extensions/wikia/FandomComMigration/FandomComMigrationHooks.class.php [hunk 2]
```diff
@@ -3,29 +3,19 @@
...
-			if ( $wgFandomComMigrationDone ) {
-				if ( !empty( $wgFandomComMigrationCustomMessageAfter ) ) {  // customized message
-					$wikiVariables['fandomComMigrationNotificationAfter'] = $parser->parse(
-						$wgFandomComMigrationCustomMessageAfter, new Title(), new ParserOptions() )->getText();
-				} else {
-					$wikiVariables['fandomComMigrationNotificationAfter'] =
-						wfMessage( 'fandom-com-migration-after' )->parse();
-				}
-			} elseif ( static::isMigrationScheduled() ) {
+			if ( static::isMigrationScheduled() ) {
```
Comment (themech) [inline extensions/wikia/FandomComMigration/FandomComMigrationHooks.class.php]: it duplicates the `if` you have above

File: extensions/wikia/FandomComMigration/FandomComMigrationHooks.class.php [hunk 3]
```diff
@@ -3,29 +3,19 @@
...
-		if ( static::isEnabled() ) {
+		if ( static::isMigrationScheduled() ) {
 			$out->addModules( 'ext.fandomComMigration' );
 		}
 
 		return true;
 	}
 
 	public static function onMercuryWikiVariables( array &$wikiVariables ): bool {
-		global $wgFandomComMigrationDone, $wgFandomComMigrationCustomMessageBefore,
```
Comment (themech) [inline extensions/wikia/FandomComMigration/FandomComMigrationHooks.class.php]: I think Growth is using this for displaying various messages on wikis, for example on https://fishing-simulator.fandom.com/wiki/Fishing_Simulator_Wiki. I think they won't be happy if you remove it

**Review by themech — CHANGES_REQUESTED** [review]

**Review by rbroering — CHANGES_REQUESTED** [review]
The help page link has no text:
`Find out more on [https://community.fandom.com/wiki/Help:Unified_Community_Platform] this Fandom help page.`
should be
`Find out more on [https://community.fandom.com/wiki/Help:Unified_Community_Platform this Fandom help page].`

**Review by themech — APPROVED** [review]

**Pull request merged**